lesswatch.bat

```diff
@@ -0,0 +1,2 @@
+@echo
+less-watch-compiler less css index.less
```

Comment:
Fully unneccessary and can be replaced with an Powershell/BASH function in your profile with an alias to call it or a custom global module.